app/views/admin/statistics_announcements/_form.html.erb

```diff
@@ -64,7 +64,11 @@
   } %>
 
   <% if statistics_announcement.new_record? %>
-      <%= render 'release_date', statistics_announcement: statistics_announcement %>
+      <%= render '/admin/shared/release_date', {
+        object: statistics_announcement.current_release_date,
```

Comment:
Could the local variable be called `current_release_date` or `release_date`? 

Generally we would use object/model in a situation where the object could be one of many classes. Otherwise, generally we assign it a variable which describes it is for clarity.

Reply:
Yes, thanks for this. Interrogated the statistics_announcement_... models a bit more and understand their inheritance better. So have amended to create_release_date as suggested.